src/NBitcoin/ConsensusOptions.cs

```diff
@@ -42,6 +44,13 @@ public class ConsensusOptions
         /// <summary>The maximum number of sigops we're willing to relay/mine in a single tx.</summary>
         public int MaxStandardTxSigopsCost { get; set; }
 
+        /// <summary>Block Height at which the node should enforce the use of <see cref="EnforcedMinProtocolVersion">.
```

Comment:
```<see cref="EnforcedMinProtocolVersion">``` is not closed

```<see cref="EnforcedMinProtocolVersion"/>``` 

same in other places

Reply:
Good spot, resolved